dev/cluster.go

```diff
@@ -183,7 +183,7 @@ func (c *Cluster) CreateAndWaitFromHttp(
 
 	for i := range objects {
 		if err := c.CreateAndWaitForReadiness(ctx, &objects[i], opts...); err != nil {
-			return fmt.Errorf("creating object: %w", err)
+			return fmt.Errorf("creating from http: %w", err)
 		}
 	}
 	return nil
```

Comment:
Hey, I know it's just a couple of lines, but I'm wondering if it's worth it to deduplicate it by extracting it into a separate method, since it appears three times. Something like:
```
func (c *Cluster) createObjects(ctx context.Context, objects []unstructured.Unstructured, source string, opts ...WaitOption) error {
	for i := range objects {
		if err := c.CreateAndWaitForReadiness(ctx, &objects[i], opts...); err != nil {
			return fmt.Errorf("creating from %s: %w", source, err)
		}
	}
	return nil
}
```

Reply:
Good catch. Thank you :)
I've incorporated your request.

Reply:
Oh, just realized that the `opts` argument in the `Create...()` functions is never used, do you want to remove it?

Reply:
Another good catch! :grinning:
I fixed it by properly passing the `opts` arg forward to `c.Waiter.WaitForReadiness(...)`.